core/src/main/java/org/apache/iceberg/GenericManifestEntry.java

```diff
@@ -88,9 +93,19 @@ public Long snapshotId() {
     return snapshotId;
   }
 
+  /** @return the data sequence number of the file */
```

Comment:
Do we need this since it is overriding an interface with the doc?

Reply:
Nope, I'll drop it.

Reply:
Removed.